### PR TITLE
Update risc0 to 1.0.0

### PR DESCRIPTION
**Pull request by kpp**
# Description
Update risc0 code to version 1.

## Linked Issues

- Related to https://github.com/chainwayxyz/citrea/issues/637
- Fixes https://github.com/chainwayxyz/citrea/issues/753
-

### CODE REVIEW

File: crates/sovereign-sdk/adapters/risc0-bonsai/src/host.rs [hunk 1]
```diff
@@ -460,17 +459,19 @@ impl<'a> ZkvmHost for Risc0BonsaiHost<'a> {
...
-                                seal: snark_receipt.snark.to_vec(),
-                                claim: receipt
-                                    .get_claim()
-                                    .expect("stark_2_snark error, receipt claim"),
-                            }),
-                            journal: risc0_zkvm::Journal {
-                                bytes: snark_receipt.journal,
-                            },
-                        };
+                        // FIXME:
```
Comment (rakanalh) [inline crates/sovereign-sdk/adapters/risc0-bonsai/src/host.rs:462]: 1. I would explain why this needs fixing to start with.
2. Maybe create an issue about this and link it here for future reference. That is, if we don't plan on fixing this within this PR.

**Review by ekrembal — COMMENTED** [review]
LGTM, But we need to test proof generation as well. @eyusufatik can work on that.

**Comment by eyusufatik** [comment]
can you update ci with versions?

**Comment by eyusufatik** [comment]
I can confirm proof production & verification works

**Review by eyusufatik — APPROVED** [review]

**Pull request merged**